php-checks/src/test/resources/checks/CountInsteadOfEmptyCheck.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+if (count($a) >= 0) { // Noncompliant
```

Comment:
I believe that we shouldn't raise an issue in this case because it should be covered by [RSPEC-3981](https://rules.sonarsource.com/php/RSPEC-3981).

Reply:
Not raised anymore now.